pkg/internal/checkup/executor/console/console.go

```diff
@@ -102,13 +110,13 @@ func SafeExpectBatchWithResponse(serialConsoleClient vmiSerialConsoleClient,
 	vmiName string,
 	expected []expect.Batcher,
 	timeout time.Duration) ([]expect.BatchRes, error) {
-	expecter, _, err := NewExpecter(serialConsoleClient, vmiNamespace, vmiName, timeout)
+	expecter := NewExpecter(serialConsoleClient, vmiNamespace, vmiName)
+	gGExpect, err := expecter.spawnConsole(timeout)
 	if err != nil {
 		return nil, err
 	}
-	defer expecter.Close()
```

Comment:
Who will now close the open connection to the VMI's serial console?

Reply:
I think it was removed by mistake and brought back in later commits - will fix

Reply:
DONE